src/pwm.rs

```diff
@@ -126,7 +196,7 @@ impl Pins<TIM4>
         PB9<Alternate<PushPull>>,
     )
 {
-    const REMAP: u8 = 0b0;
+    const REMAP: u8 = 0b00;
```

Comment:
I think this change is wrong.  `TIM4_REMAP` is, afaict, only one bit.

Zero means PB6,7,8,9 ånd One means PD12,13,14,15

Reply:
I mean, the change is harmless enough, but it could lead somene to think that the value was wider than one bit.

Reply:
Good catch ! I will push a fixup.